Donation-platform/src/test/java/com/google/DbCallsTest.java

```diff
@@ -237,65 +227,17 @@ public void updateUserTest() throws Exception {
   public void getCharityByIdTest() throws Exception {
     dbCalls.addCharity(CHARITY_A, LINK_A, IMGSRC_A, CATEGORIES_A, DESCRIPTION_A);
     Charity expectedCharity = dbCalls.getCharityByName(CHARITY_A);
-    Key expectedCharityId = expectedCharity.getId();
+    Long expectedCharityId = expectedCharity.getId();
     Charity actualCharity = dbCalls.getCharityById(expectedCharityId);
-    assertEquals(expectedCharity.getId(), actualCharity.getId());
+    assertEquals(expectedCharity.getName(), actualCharity.getName());
   }
   // Test checks if function gets Tag by ID from database.
   @Test
   public void getTagByIdTest() throws Exception {
     dbCalls.addTag(TAG_A, TRENDINGSCORE_A, IMGSRC_A);
     Tag expectedTag = dbCalls.getTagByName(TAG_A);
-    Key expectedTagId = expectedTag.getId();
+    Long expectedTagId = expectedTag.getId();
     Tag actualTag = dbCalls.getTagById(expectedTagId);
     assertEquals(expectedTag.getId(), actualTag.getId());
   }
-  // Test checks if we can get Tag objects from collection of Ids.
```

Comment:
Why are these tests removed rather than changed to use the new IDs?

Reply:
These were helper functions to help me convert entities to classes. Objectify does this automatically hence, I no longer need to handle such cases.

Reply:
Ok sg